### PR TITLE
Package describe should always succeed.

### PR DESCRIPTION
**Pull request by jsancio**

### CODE REVIEW

File: cosmos-integration-tests/src/main/scala/com/mesosphere/cosmos/PackageDescribeSpec.scala [hunk 1]
```diff
@@ -64,15 +64,20 @@ final class PackageDescribeSpec
     }
 
     "when requesting a v2 response" - {
-      "fails to describe helloworld v4 w/ updates when requesting v2 describe response" in {
+      "should succeed to describe helloworld v4 when requesting v2 describe response" in {
```
Comment (cruhland) [inline cosmos-integration-tests/src/main/scala/com/mesosphere/cosmos/PackageDescribeSpec.scala]: Nitpick: "should succeed to describe" -> "successfully describes"

File: cosmos-integration-tests/src/main/scala/com/mesosphere/cosmos/PackageDescribeSpec.scala [hunk 2]
```diff
@@ -149,7 +154,7 @@ final class PackageDescribeSpec
       )
     )
 
-    val Right(actualContent) = parse(response.contentString)
```
Comment (cruhland) [inline cosmos-integration-tests/src/main/scala/com/mesosphere/cosmos/PackageDescribeSpec.scala]: Why did this change?

File: cosmos-integration-tests/src/main/scala/com/mesosphere/cosmos/PackageDescribeSpec.scala [hunk 3]
```diff
@@ -162,7 +167,7 @@ final class PackageDescribeSpec
       rpc.v1.model.DescribeRequest("helloworld", version)
     )
     assertResult(Status.Ok)(response.status)
-    val Right(packageInfo) = parse(response.contentString)
```
Comment (cruhland) [inline cosmos-integration-tests/src/main/scala/com/mesosphere/cosmos/PackageDescribeSpec.scala]: Why did this change?

File: cosmos-server/src/main/scala/com/mesosphere/cosmos/converter/Response.scala [hunk 4]
```diff
@@ -79,45 +79,39 @@ object Response {
...
-          pkg.config,
-          pkg.command
-        )
+      val packagingVersion = pkg match {
+        case _: universe.v3.model.V2Package =>
+          universe.v3.model.V2PackagingVersion
+        case _: universe.v3.model.V3Package =>
+          universe.v3.model.V3PackagingVersion
+        case _: universe.v4.model.V4Package =>
+          universe.v3.model.V3PackagingVersion
```
Comment (cruhland) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/converter/Response.scala]: Should this be `V4PackagingVersion`?

Comment (cruhland) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/converter/Response.scala]: Based on this change, it looks like we are now okay with dropping information in responses. Should the `README` or `CONTRIBUTING` files be updated?

Do we have a ScalaCheck test that verifies this conversion works on any randomly-generated `PackageDefinition`?

**Comment by cruhland** [comment]
@jsancio there seems to be a problem with the `dcos-launch` script for the integration tests, any ideas?

File: cosmos-server/src/main/scala/com/mesosphere/cosmos/converter/Response.scala (hunk 4 above)
Comment (jsancio) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/converter/Response.scala]: If we don't do this we need to ask CLI and UI to update to the new version of the response in DCOS-15084.

**Comment by jsancio** [comment]
Going with a different solution.

**Pull request closed without merging**